packages/amazon-sumerian-hosts-babylon/src/Babylon.js/awspack/LexFeature.js

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import {LexFeature as CoreLexFeature} from '@amazon-sumerian-hosts/core';
+
+/**
+ * @extends core/LexFeature
+ * @alias babylonjs/LexFeature
+ */
+class LexFeature extends CoreLexFeature {
+  _setupAudioContext() {
```

Comment:
Related to the dependency injection comment. I'm fine removing this function and just accept user passed in AudioContext object if we can assume users able to handle the AudioContext creation by themselves.

Reply:
Wherever we can we should save users from having to do any configuration, plumbing, or logic that we can take care of for them. I can't think of any customer benefit to requiring them to get their own reference to the singleton audio context of the engine they're using and explicitly pass it along to us.

I think Preston's comment was only about how to ensure we provide the most accurate documentation for this method.

Reply:
This particular comment was indeed about the documentation and use of jsdoc.

runping is referring to an earlier comment about not using the babylon object as a global. I agree with kris that this is not worth forcing customers thinking about dependencies like audiocontext, but I was envisioning that the babylon host framework does this. Said another way, is it an easy refactor to only refer to the static BABYLON once in the codebase instead of doing it in many different places?